src/fsharp/CompileOps.fs

```diff
@@ -4624,8 +4624,7 @@ let RequireDLL (ctok, tcImports:TcImports, tcEnv, thisAssemblyName, m, file) =
     let tcEnv = (tcEnv, asms) ||> List.fold (fun tcEnv asm -> AddCcuToTcEnv(g,amap,m,tcEnv,thisAssemblyName,asm.FSharpViewOfMetadata,asm.AssemblyAutoOpenAttributes,asm.AssemblyInternalsVisibleToAttributes)) 
     tcEnv,(dllinfos,asms)
 
-       
-let paketPrefix = "paket: "
```

Comment:
Yeah I should revert that part. The thing I wanted to support was basically a switch to just activate the paket references from the ambient paket.dependencies that is already existing in the repo. Basically I thought
    
    #r "paket: "

Looks weird, so let's make it 

    #r "paket" 

But that's something for the language design part. I will use the first version for now.